app/src/main/res/values/dimens.xml

```diff
@@ -262,5 +262,7 @@
     <!-- Category Edit Button -->
     <dimen name="category_edit_button_hidden_margin_start">16dp</dimen>
     <dimen name="category_edit_button_hidden_margin_end">8dp</dimen>
+    <dimen name="custom_category_button_text_size_phone">22sp</dimen>
+    <dimen name="custom_category_button_text_size_tablet">34sp</dimen>
```

Comment:
So we actually have a separate `dimens` file for tablets (and also landscape). It's set up the same way as the layout files, in directories named with `sw600dp`, `land`, etc. That way you only have to have one style that will pull dimens from the correct resource set.

Reply:
I knew there had to be a way to do that, I was looking for different styles files.  I really need to work with different screen formats more